Model/Entities/Entities.swift

```diff
@@ -161,6 +177,11 @@ struct URLContent {
             return mime
         }
     }
+
```

Comment:
This currently using the ZIM file's modification date time to create the eTag, which I think a good enough solution for our purposes.

Reply:
My thinking here was, that I looked at what headers are returned in Safari / Caddy when webm is played.
So I wanted to have the very same ones, to make sure it works. One of them was this eTag, which can be generated based on the date the ZIM file was changed.
We can omit this if you think it is not needed.

Reply:
No harm in keeping it IMO. ETag is supposed to be a uniq version ID for a resource and this implementation is OK in this regard since all resources are single-version made at the same time in a ZIM.
Still it feels like abusing the header to have the same for all resources 😀
Mandates a comment explaining this I think